src/createServer.js

```diff
@@ -1,3 +1,33 @@
-// Write code here
-// Also, you can create additional files in the src folder
-// and import (require) them here
+const http = require('http');
+const { convertToCase } = require('./convertToCase/convertToCase');
+const { parseURL } = require('./parseURL');
+const { validateData } = require('./validateData');
+const { sendResponse } = require('./sendResponse');
+
+const createServer = () => {
+  return http.createServer((req, res) => {
+    const { stringCase, queryString } = parseURL(req.url, req.headers.host);
+    const validation = validateData(queryString, stringCase);
+
+    res.writeHead(200, { 'Content-Type': 'application/json' });
+
+    if (!validation.errors.length) {
+      const result = {
+        ...convertToCase(queryString, stringCase),
+        originalText: queryString,
+        targetCase: stringCase,
+      };
+
+      sendResponse(res, 200, result);
+
+      return;
+    }
+
+    if (validation.errors.length) {
```

Comment:
you already check it in 14 like
if thick check === false `If (!validation.errors.length)`  it means that `validation.errors.length` === true )
```suggestion
```